README.md

```diff
@@ -47,6 +47,9 @@ pride add network-component backend-api
 # Now you do some work and realize you also need another module (with an absolute URL)
 pride add https://github.com/myself/myproject
 
+# To add a mercurial module, use --repo-type
+pride add --repo-type hg mercurial-project
+
```

Comment:
As far as I can see, `HgVcsSupportFactory.canSupport()` is only used in [InitAction](https://github.com/prezi/pride/blob/master/pride/src/main/java/com/prezi/pride/cli/commands/actions/InitAction.java#L83). But this relates to `AddAction`. I think the problem is in
[AddCommand.getModulesToAdd](https://github.com/prezi/pride/blob/master/pride/src/main/java/com/prezi/pride/cli/commands/AddCommand.java#L71). It picks up `VcsSupport` based on either an explicit repo type or a default one (git).

I was going to solve that one in the next PR. But I can try to solve it here if you want. It would probably mean changing `AddCommand`.